modules/qtwidgets/src/filepathlineeditqt.cpp

```diff
@@ -90,6 +90,9 @@ FilePathLineEditQt::FilePathLineEditQt(QWidget* parent)
     connect(this, &QLineEdit::editingFinished, [this]() {
         setCursorToEnd();
         const auto path = utilqt::toPath(text().trimmed());
+
+        // FIXME: handle case where new path is not valid. Should the internal path be updated? Does
+        // the line edit need updating?
```

Comment:
I think the actually error handing should happen in the property not in the widget. So I don't think we should do anything here